nexus_motion_planner/test/test_request.test.py

```diff
@@ -179,6 +187,34 @@ def generate_test_description():
         "publish_transforms_updates": True,
     }
 
+    # Load ros2_control controller (so joint states are published)
+    # This needs to be loaded in tandem with the MoveIt controller manager
+    ros2_controllers_path = os.path.join(
+        get_package_share_directory(moveit_config_package),
+        "config",
+        "ros2_controllers.yaml",
+    )
+    ros2_control_node = Node(
+        package="controller_manager",
```

Comment:
Add `<test_depend>` for ros2_control pkgs.

Reply:
51f34f4c6170d6cc978e2be34190e8efee8833f8